### DISCUSSION TITLE
Render tooltip inside portal?

### DISCUSSION THREAD

**Discussion by viveleroi** [ReactTooltip/react-tooltip#1164] [Q&A] [answered]
We have frames and transform/scale elements which can interfere with `position: absolute/fixed`. Is there a way to render the tooltip inside a portal? We already have portal elements handy but I'm not seeing a way to customize the actual renderer.

**Comment by gabrieljablonski** [comment]
Sorry, I saw you opened the feature request (#1163) but didn't have the time this week to look into this.

Can you show me a quick example of a component of yours that uses portals, or any other library you've seen that does this well?

That might help me throw a quick example together with `react-tooltip` so we can have a usable example ready until we decide if we're implementing it internally.

**Reply by viveleroi** [reply]
Oh, I forgot I had opened a feature request - my bad for doubling up. we've been pushing several libraries to their limits and I sometimes lose track. I'll reply there

**Comment by danielbarion** [comment, marked as answer]
Will be implemented on v6 based on the issue created.
